README.md

```diff
@@ -1864,6 +1864,11 @@ The following extractors use this feature:
 #### digitalconcerthall
 * `prefer_combined_hls`: Prefer extracting combined/pre-merged video and audio HLS formats. This will exclude 4K/HEVC video and lossless/FLAC audio formats, which are only available as split video/audio HLS formats
 
+#### mixchmovie
+
+* `max_comments`: Maximum number of comments to extract - default is `120`. The final amount of comments is less than the limit because gifts are filtered.
```

Comment:
The default should be infinite.
I also don't like the ambiguity of gifts not being included. Maybe we should have this be `max_comments_pages`?